tasks.rb

```diff
@@ -391,6 +392,7 @@ def create_hpxmls
     'base-hvac-none.xml' => 'base.xml',
     'base-hvac-portable-heater-gas-only.xml' => 'base.xml',
     'base-hvac-programmable-thermostat.xml' => 'base.xml',
+    'base-hvac-onoff-thermostat-deadband.xml' => 'base-hvac-air-to-air-heat-pump-1-speed.xml',
```

Comment:
This will be a challenge indeed. Not immediately sure what to suggest. At a minimum, we need to manually run simulations for different system types, thermostat setpoint input types, etc. if that hasn't already been done.

Reply:
Let's just go with two test files -- one for simple thermostat setpoints (which you already have) and one with 8760 scheduled setpoints.